index.js

```diff
@@ -79,4 +91,98 @@ function canParse(path) {
   return true;
 }
 
+async function reportCoverageComment(
+  linesMissingCoverageByFile,
+  totalFinds,
+  totalHits,
+  coverageString
+) {
+  const gitHubToken = core.getInput('github-token').trim();
+  const projectNameInput = core.getInput('project_name');
+  if (gitHubToken !== '' && github.context.eventName === 'pull_request') {
+    try {
+      const octokit = github.getOctokit(gitHubToken);
+      const sha = github.context.payload.pull_request.head.sha;
+      const shaShort = sha.substr(0, 7);
+
+      const summary = `Summary coverage rate: \n  ${coverageString}% (${totalHits} of ${totalFinds} lines)`;
+
+      const missedLines = linesMissingCoverageByFile
+        .map((line) => `  ${line}`)
+        .join('\n');
+
+      const projectName = projectNameInput || github.context.workflow;
+      const missedLinesString =
+        linesMissingCoverageByFile.length === 0
+          ? ''
+          : `Missed lines:\n${missedLines}`;
+
+      const coverageMet = linesMissingCoverageByFile.length === 0;
+      const coverageStatus = coverageMet
+        ? `✅ Minimum coverage met`
+        : `❌ Minimum coverage treshold not met!`;
```

Comment:
question: would this only be the case if the minimum coverage is 100? It seems like passing in `isValidBuild` into this function would be a safer way of setting this status